### PR TITLE
Add christopher file

### PR DESCRIPTION
**Pull request by chrizzit**

### CODE REVIEW

File: christopher.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Welcome To SWFL Coders!</title>
+</head>
+<body>
+  <h2>Coding is so cool!</h2>
```
Comment (digitaldrk) [inline christopher.html]: Keep it semantic please use an `<h1>`

Comment (digitaldrk) [reply christopher.html]: Change to h1 please

File: christopher.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Welcome To SWFL Coders!</title>
+</head>
+<body>
+  <h1>Coding is so cool!</h1>
```
Comment (digitaldrk) [inline christopher.html]: Great job!

Comment (johndavidpena) [reply christopher.html]: so semantic, awesome bro

**Pull request merged**